src/blog/2024/09/node-red-version-control-with-snapshots.md

```diff
@@ -0,0 +1,238 @@
+
+![Image showing the option to switch to the Snapshots tab in FlowFuse.](./images/option-to-switch-to-snapshots-tab.png){data-zoomable}
+*Image showing the option to switch to the Snapshots tab in FlowFuse.*
+
+3. Switch to the **“Snapshots”** tab by selecting it from the instance management options. Here, you will find options to create a new snapshot called **"Create Snapshot"**.
+
+![Image showing the button to create a snapshot in FlowFuse.](./images/button-to-create-snapshot.png){data-zoomable}
+*Image showing the button to create a snapshot in FlowFuse.*
+
+4. Click on the button to **create snapshot**. You will be prompted to enter a **name and description** for the snapshot, helping you identify it later. There’s also an option to set this snapshot as Device Target Snapshot — enable this checkbox if needed.
+
+![Image showing the form to provide a name and description for the snapshot.](./images/form-to-give-snapshot-name-desc.png){data-zoomable}
+*Image showing the form to provide a name and description for the snapshot.*
+
+5. After creating the snapshot, you will receive a confirmation message indicating that the snapshot has been successfully created.
+
+![Image showing a list of created snapshots with details in FlowFuse.](./images/snapshot-in-the-list.png){data-zoomable}
+*Image showing a list of created snapshots with details in FlowFuse.*
+
+Once a snapshot is created, it will be visible in a list format. Each snapshot will display details such as the name, description, creator, and creation date, along with a three-dot icon. 
+
+![Image showing available options for managing snapshots in FlowFuse.](./images/options-available-for-instance.png){data-zoomable}
+*Image showing available options for managing snapshots in FlowFuse.*
+
+Clicking on the icon will open different options to manage and operate the snapshot.
+
+### Creating Snapshot for Device Instances when assigned to Cloud Instance
+
+To create a snapshot for the device instances when that device is assinged to cloud instance:
+
+1. Go to **Devices** by clicking on the **Devices** option in the sidebar, and then click on the specific device.
+2. Once clicked, a similar interface will open as with instances, allowing you to manage and monitor its settings and configuration. Ensure that the device has developer mode enabled.
+
+![Image showing the 'Create Snapshot' option in the Developer mode tab](./images/option-to-create-device-snapshot.png
+){data-zoomable}
+*Image showing the 'Create Snapshot' option in the Developer mode tab*
+
+3. Switch to the **Developer Mode** tab by clicking on the **Developer Mode** option at the top. In the Developer Mode section, you will see the **Create Snapshot** button. Click on it to create a snapshot, then enter the details such as the name and description. If needed, set it as the device target snapshot, and then click **Create**.
```

Comment:
The way this reads to me is:

1. I must go to the instance -> devices view
2. Switch it into developer mode
3. Create a snapshot.

This is both counter productive and a little misleading.

If a device is in fleet mode, it WILL be running the "active snapshot" specified on the devices overview. 

I suspect we need to somehow make it clear that this is really only necessary if the device is in developer mode as it may have been modified. Perhaps something as simple as stating "typically, the device will be in fleet mode and will be running the flows specified by the target snapshot, however, if the device is in developer mode it may have been modified, in which case, you can take a snapshot directly from the device using the developer tab - here's how..."